ForgeRock Cloud APIs.postman_collection.json

```diff
@@ -848,7 +844,7 @@
 								},
 								{
 									"key": "Location",
-									"value": "https://ui-spark20.forgeblocks.com/hosted/login?goto=https%3A%2F%2Fopenam-spark20.forgeblocks.com%2Fopenam%2Foauth2%2Fauthorize%3Fclient_id%3D070462ae12ec98a1e05e65e8c2148802%26response_type%3Dcode%26redirect_uri%3Dhttp%253A%252F%252Flocalhost%253A9080%252Fcallback%26scope%3Dapi.forgecloud.com%253Auser.read%2520openid&realm=/",
+									"value": "https://ui-spark20.forgeblocks.com/hosted/login?goto=https%3A%2F%2Fopenam-spark20.forgeblocks.com%2Fopenam%2Foauth2%2Fauthorize%3Fclient_id%3D070462ae12ec98a1e05e65e8c2148802%26response_type%3Dcode%26redirect_uri%3Dhttp%253A%252F%252Flocalhost%253A9080%252Fcallback%26scope%3Duser.read%2520openid&realm=/",
```

Comment:
variable?

Reply:
atm, there's a variable set for {{oauth2Url}}, which isn't working. I'd like to split that up, into {{tenant-name}} and {{domain}}

Reply:
Updated in new PR